src/io_meandata.F90

```diff
@@ -257,17 +257,9 @@ subroutine ini_mean_io(ice, dynamics, tracers, partit, mesh)
     end if
 CASE ('m_snow    ')
     if (use_ice) then
-    call def_stream(nod2D, myDim_nod2D, 'm_snow',   'snow height  per unit area',      'm',      ice%data(3)%values(1:myDim_nod2D),     io_list(i)%freq, io_list(i)%unit, io_list(i)%precision, partit, mesh)
+    call def_stream(nod2D, myDim_nod2D, 'm_snow',   'snow height',                     'm',      ice%data(3)%values(1:myDim_nod2D),     io_list(i)%freq, io_list(i)%unit, io_list(i)%precision, partit, mesh)
     end if
-CASE ('h_ice     ')
```

Comment:
Is the removal of h_ice and h_snow intended?

Reply:
Strange ... I did not touch gen_modules_clock.F90 and io_meandata.F90 at these lines.